install/install.bash

```diff
@@ -0,0 +1,105 @@
+#!/bin/bash
+
+if [ "$(which aws)" == "" ]; then
+    echo "Couldn't find the aws-cli ('aws') on your path - is it installed? http://docs.aws.amazon.com/cli/latest/userguide/installing.html"
+    exit 1
+fi
+
+if [ "$(which jq)" == "" ]; then
+    echo "Couldn't find 'jq' on your path - is it installed?"
+    exit 1
+fi
+
+lambda_zip=$(ls ../build/distributions/*.zip 2> /dev/null | head -1)
+
+if [ "${lambda_zip}" == "" ]; then 
+    echo "Couldn't find the lambda zip to upload in '../build/distributions/' (do you need to build it?)"
+    exit 1
+fi  
+
+read -p "Enter Snowplow collector URL: " collector_url
+if [[ ! "${collector_url}" =~ ^http://.* ]]; then
+    collector_url="http://${collector_url}"
+fi
+
+read -p "Enter S3 bucket name to monitor events for: " s3_bucket_name
+
+if [ "${s3_bucket_name}" == "" ]; then
+    echo "Bucket name is required!"
+    exit 1
+else 
+    avail_buckets="$(aws s3api list-buckets --query 'Buckets[].Name' | jq -r -M .[] | grep ${s3_bucket_name})"
+    if [ "${avail_buckets}" == "" ]; then
+        echo "Couldn't find bucket '${s3_bucket_name}', does it exist?"
+        exit 1
+    fi    
+fi  
+
+account_id="$(aws ec2 describe-security-groups --group-names 'Default' --query 'SecurityGroups[0].OwnerId' --output text)"
+
+echo "Using account id '${account_id}'"
+
+echo "Creating policy..."
+
+policy_result="$(aws iam create-policy --policy-name snowplow-s3-source --policy-document file://perms.json)"
```

Comment:
The file:// will break if the script is run like `/foo/bar/install.bash` instead of `cd /foo/bar && ./install.bash`. It's always safer for Bash scripts to start with a `cd` into the exact path of the executing script before doing any work.